anago

```diff
@@ -200,6 +200,46 @@ check_acls () {
 }
 
 ###############################################################################
+# Ensures all registries that will be used during both mock and --nomock
+# runs allow write access so we don't fall over later
+# @param registries - A space separated list of registries
+#
+ensure_registry_acls () {
```

Comment:
I'm still getting access denied on a mock build, even though I'm authenticated as the production user:

```
- push gcr.io/kubernetes-release-test/kube-controller-manager-s390x:v1.6.8
The push refers to a repository [gcr.io/kubernetes-release-test/kube-controller-manager-s390x]
63e22cdd38b0: Preparing
9822fe240452: Preparing
denied: Unable to access the repository; please check that you have permission to access it.
```